packages/next/src/shared/lib/dynamic.tsx

```diff
@@ -40,10 +40,6 @@ export type DynamicOptions<P = {}> = LoadableGeneratedOptions & {
   loader?: Loader<P> | LoaderMap
   loadableGenerated?: LoadableGeneratedOptions
   ssr?: boolean
-  /**
```

Comment:
This is indended to be dropped

Reply:
It was deprecated before, the option was already removed. We just drop the type now.